doc/explanation/storage.md

```diff
@@ -28,7 +28,7 @@ Storage location         | Directory | Btrfs    | LVM      | ZFS      | Ceph RBD
 :---                     | :-:       | :-:      | :-:      | :-:      | :-:      | :-:
 Shared with the host     | &#x2713;  | &#x2713; | -        | &#x2713; | -        | -
 Dedicated disk/partition | -         | &#x2713; | &#x2713; | &#x2713; | -        | -
-Loop disk                | &#x2713;  | &#x2713; | &#x2713; | &#x2713; | -        | -
+Loop disk                | -         | &#x2713; | &#x2713; | &#x2713; | -        | -
```

Comment:
What do you think of changing "Loop disk" here to "Loop disk (in `<LXD_lib_dir>/disks/`)"?

Reply:
Or expanding the Loop description description below covering where the loop file actually resides?